src/plugins/recordTypes/media/advancedSearch.js

```diff
@@ -0,0 +1,63 @@
+export default (configContext) => {
+  const {
+    OP_EQ,
+    OP_CONTAIN,
+    OP_RANGE,
+  } = configContext.searchOperators;
+
+  const {
+    defaultAdvancedSearchBooleanOp,
+    extensions,
+  } = configContext.config;
+
+  return {
+    op: defaultAdvancedSearchBooleanOp,
+    value: [
+      {
+        op: OP_EQ,
+        path: 'ns2:media_common/creator',
+      },
+      {
+        op: OP_RANGE,
+        path: 'ns2:media_common/dateGroupList/dateGroup',
+      },
+      {
+        op: OP_CONTAIN,
+        path: 'ns2:media_common/identificationNumber',
+      },
+      {
+        op: OP_EQ,
+        path: 'ns2:media_common/languageList/language',
+      },
+      {
+        op: OP_EQ,
+        path: 'ns2:media_common/publisher',
+      },
+      {
+        op: OP_EQ,
+        path: 'ns2:media_common/rightsHolder',
+      },
+      {
+        op: OP_CONTAIN,
+        path: 'ns2:media_common/source',
+      },
+      {
+        op: OP_CONTAIN,
```

Comment:
Should be `OP_EQ`. Remember to change the operator for any field you change to autocomplete/controlled.